addon/components/new-osf-navbar/template.hbs

```diff
@@ -0,0 +1,75 @@
+<div class="new-osf-navbar-styling">
+    <nav class="navbar navbar-inverse" id="navbarScope">
+        <div class="container">
+            <div class="navbar-header">
+                {{!Toggle navigation button for xs screens}}
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".secondary-navigation" {{action 'closeSearch'}}>
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
```

Comment:
I thought the "sr-only" class did a similar thing.  I'll remove it and add an aria-label.